bin/env_variables_all.txt

```diff
@@ -79,6 +86,9 @@ ELASTIC_SEARCH_HOSTS=http://localhost:9200
 ELASTIC_SEARCH_PORT=9200
 ELASTIC_SEARCH_SCHEME=http
 IDAM_API_URL_BASE=http://localhost:5000
+IDAM_API_BASE_URL=http://localhost:5000
+IDAM_OIDC_URL=http://idam-api:5000
+IDAM_STUB_SERVICE_NAME=http://idam-api:5000
```

Comment:
Do we need this specified three more times in addition to the 2 other legacy values below?  

NB: most repos (AAC, Case-Docs, CPO, Data-store) seam to have a good enough default for `IDAM_OIDC_URL` in their respective application property files to cope with running locally. 

And setting: 'IDAM_STUB_SERVICE_NAME' may only be required for those choosing to run against an IDAM stub rather than the docker version.

Reply:
FYI: if they do need to be retained this file is currently in alphabetical order.